src/xtgeo/common/sys.py

```diff
@@ -618,7 +620,11 @@ def _detect_fformat_by_contents(self, details: Optional[bool] = False):
         # ...
         # The signature here is one float in first line with values 1.0; one string
         # in second line; and 3 or 4 items in the next (sometimes RKB is missing)
-        xbuf = buf.decode().split("\n")
+        try:
+            xbuf = buf.decode().split("\n")
+        except UnicodeDecodeError:
```

Comment:
This bugfix should be a separate commit.

Reply:
OK. I will rename this commit message.

This was needed as some tests would fail otherwise, after the changes in other modules

Reply:
Ups separate 'commit' yes, not PR!. Wait... will fix and I will do a new push